Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+during the design or architecture phase of new services, when an initial data flow diagram for the service has been created.
+
+== What to run through RRAs? ==
+
+RRAs can only be used for services. Any project belongs to or provides a service. Any component, piece of code or data
+belongs to a service. Find which service it belongs to and RRA that service, not the specific piece of code or data.
+
+For large services, these can be split into multiple smaller services or sub-services. A choice has to be made by the
+security engineer running the RRA. If the sub-services belong to different teams, it is a strong indicator that
+multiple RRAs should be ran.
+
+== What to focus on during RRAs? ==
+
+* '''Impact assessment'''. The RRA is the authority for impact levels and these are paramount.
+* '''Rationales'''. Anyone should understand why you've set a specific impact level by reading 
+* Getting '''value for the team'''. The team needs to understand what is most important to protect.
+
+What to '''NOT''' focus on:
+* Gathering security controls, figuring out how effective they are, etc. Don't do that! This information may be recorded if it comes up but do not focus on it.
+* Likelihood, Security provided by service. Don't spent much time there! These are "10, 000 feet approximations" and part of a larger risk calculation mechanism.
+
+== Guided process: Running your RRA in ~30 minutes ==
+
+This is a guided example of how to run an RRA.
+
+{| class="wikitable"
+|-
+! <span style="color:gray;">Time management and control</span>
+|-
+|
+You will be responsible for the time distribution and ensure 30 minutes are not exceeded if possible.
+
+This means that you will sometimes have to cut a discussion short. An incomplete RRA due to time being exceeded has no
+value. Do not hesistate to take control and enforce your own discussion topics. Examples of when you'll need to be
+assertive and regain control:
+
+* The discussion languishes (>1 minute) around "how to mitigate this very issue" or "we're doing X to ensure this never happens".
+* The discussion focuses on process instead of filling the form.
+* The discussion about how the service works takes forever (>15 minutes) and the owner has to lookup every single detail.
+
+A good tip is to '''reserve 50-60 minutes''' of RRA time in the calendar, and plan to run the RRA for only '''30
+minutes'''.
+This leaves you with some room for error, and handle services that weren't well understood by their owners.
+|}
+
+=== Getting started (pre-work) ===
+
+* Ensure no previous RRA exist, else re-use the previous RRA.
+* Create a copy of the [https://drive.google.com/open?id=160V89R-VdIe1AEHcT_sX89UToV2gv8mLshhoia2-6mM template] and move it to the correct directory (or your personal Google drive if you're testing the RRA process).
+* Invite 1 or 2 members related to the service with enough technical knowledge, ensure they will bring a diagram of data
```

Comment:
Id say lead engineers. it really depends on the project, to find out who has best visibility. I added some examples (product owners, lead engineers, etc.)
when the product owner has the best visibility its structurally ideal - but it's not really always the case, or who is the product owner is defined differently by different teams, which leads to confusion. :|